src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/licence/Licence.kt

```diff
@@ -53,7 +53,7 @@ data class Licence(
 
   @Schema(description = "The date and time that the licence was first created", example = "2022-08-24T11:30:33")
   @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss")
-  val createdDateTime: LocalDateTime,
+  val createdDateTime: LocalDateTime?,
```

Comment:
So this value is used in multiple places and should always be set, I believe. Making it nullable for some edge case?